Submitted By: Jim Gifford (patches at jg555 dot com)
Date: 2003-08-16
Initial Package Version: 0.93
Origin: Grub CVS
Description: Fixes GCC 3.3 Compile Errors

diff -Naur grub-0.93.orig/stage2/fsys_reiserfs.c grub-0.93/stage2/fsys_reiserfs.c
--- grub-0.93.orig/stage2/fsys_reiserfs.c	2002-11-29 17:46:01.000000000 +0000
+++ grub-0.93/stage2/fsys_reiserfs.c	2003-08-16 21:15:34.000000000 +0000
@@ -112,7 +112,7 @@
   /* offset in the log of where to start replay after a crash */
   __u32 j_first_unflushed_offset;
   /* mount id to detect very old transactions */
-  __u32 long j_mount_id;
+  __u32 j_mount_id;
 };
 
 /* magic string to find desc blocks in the journal */
